src/mono/wasm/runtime/loader/globals.ts

```diff
@@ -39,7 +51,10 @@ export function setLoaderGlobals(
         config: { environmentVariables: {} }
     });
     Object.assign(runtimeHelpers, {
+        mono_wasm_bindings_is_ready: false,
+        javaScriptExports: {} as any,
         config: globalObjects.module.config,
+        enablePerfMeasure: true,
```

Comment:
did you mean to enable this here?

Reply:
It almost doesn't matter because the normalizeConfig will replace it. But yeah, removed.

Reply:
It gets automatically disabled later during startup, as I understand it